test/e2e-test-application/e2e/test3/0-setuptests/setup-test.spec.js

```diff
@@ -0,0 +1,11 @@
+describe('Navigation', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.env('url'));
+  });
+  describe('iFrame check', () => {
+    it('Check if iFrame exists in running application.', () => {
+      cy.wait(5000);
+      cy.get('iframe');
+    });
+  });
+});
```

Comment:
(q) If it was just a setup test for the development, remove it. If not, all good :).

Reply:
This is the required test for the process, described in the issue :).

Install example --> start webserver --> check if an iframe exists